### PR TITLE
{lib}[intel/2017a] libiconv v1.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2088.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a91473cb9583da486afec971d6fcb90d for a full test report.

File: easybuild/easyconfigs/l/libiconv/libiconv-1.15-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'libiconv'
+version = '1.15'
+
+homepage = 'https://www.gnu.org/software/libiconv'
+description = "Libiconv converts from one character encoding to another through Unicode conversion"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libiconv/libiconv-1.15-intel-2017a.eb]: GCCcore?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2151.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2ece187500b32e00a9f060e6cee346c6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2523.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/29e16fdd77af7adaf2e6e411986ab2cf for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**